experimental/dds/tree2/src/shared-tree-core/editManager.ts

```diff
@@ -188,7 +188,6 @@ export class EditManager<
 		// Record the sequence number of the branch's base commit on the trunk
 		const trunkBase = { sequenceNumber: trackBranch(branch) };
 		// Whenever the branch forks, register the new fork
-		const offFork = branch.on("fork", (f) => this.registerBranch(f));
```

Comment:
This comment should also be removed, right?

Reply:
Absolutely, good catch thank you.